Doc/library/math.rst

```diff
@@ -812,6 +717,69 @@ Special functions
    .. versionadded:: 3.2
 
 
+Number-theoretic functions
+--------------------------
+
+For backward compatibility, the :mod:`math` module provides also aliases of
+the following functions from the :mod:`math.integer` module:
```

Comment:
Please mention there, that these aliases are soft-deprecated.

BTW, why this section was moved?

Reply:
Maybe it is, if we want to communicate users this renaming.  Though, it's important only for those who read docs in order...  Up to you.